sources/api/prairiedog/Cargo.toml

```diff
@@ -15,12 +15,14 @@ constants = { path = "../../constants", version = "0.1" }
 log = "0.4"
 nix = "0.26"
 models =  { path = "../../models", version = "0.1" }
```

Comment:
This is the one that needs to be removed.

```suggestion
```

Reply:
Fixed